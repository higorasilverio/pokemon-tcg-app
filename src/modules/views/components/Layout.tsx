import styles from '@/styles/Layout.module.scss'
import { ReactNode } from 'react'

const Layout = ({ children }: { children: ReactNode }) => {
  return (
    <div className={styles.layout} data-cy="layout">
      {children}
    </div>
  )
}

export default Layout
