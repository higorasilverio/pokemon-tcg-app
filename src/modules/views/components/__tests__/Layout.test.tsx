import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'
import Layout from '../Layout'

describe('Layout', () => {
  it('renders unchanged', () => {
    const { container } = render(<Layout>Test</Layout>)

    expect(container).toMatchSnapshot()
  })

  it('should render correctly', () => {
    render(<Layout>Test</Layout>)

    const text = screen.getByText(/test/i)

    expect(text).toBeInTheDocument()
  })
})
