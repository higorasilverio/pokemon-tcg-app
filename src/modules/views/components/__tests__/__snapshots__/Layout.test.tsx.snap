// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders unchanged 1`] = `
<div>
  <div
    class="layout"
    data-cy="layout"
  >
    Test
  </div>
</div>
`;
